web_app/dist/main.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+        <link rel="icon" type="images/png" href="images/icon.png" />       
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>TaskViewer</title>
+        <meta name="description" content="Schedule tasks, without the clutter">
+        <link rel="stylesheet" href="main.css">
+    </head>
+
+    <body>
+
+        
+    <div class="navbar">
+        <img id="logo" src="images/iconSmall.png" alt="TaskViewer Logo">
+        <a href="#Home">Home</a>
+        <a href="#Sort">Sort</a>
+        <a href="#Filter by">Filter by</a>
+        <button class="button button5">+</button>
+        <input type="text" placeholder="Search..">
+    </div>
+        
+    <div id="main">
+        <h1>Introduction</h1>
```

Comment:
Discussed offline, I don't think this is the correct place to put an introduction/ all the other stuff